CHANGES_UNRELEASED.md

```diff
@@ -3,3 +3,30 @@
 # Unreleased Changes
 
 [Full Changelog](https://github.com/mozilla/application-services/compare/v0.42.2...master)
+
+## Android
+
+### What's new
+
+- Exposed `stroage::bookmarks::erase_everything`, which deletes all bookmarks without affecting      history, through FFI.
+
+## FxA Client
+
+### What's new
+
+Android: Add ability to get an OAuth code using a session token via the `authorizeOAuthCode` method.
+
+## Push Client
+
+### What's new
```

Comment:
you should remove the stuff above, that is already in the real changelog

Reply:
Ah, merge artifact. Sorry.